CONTRIBUTING.md

```diff
@@ -76,191 +89,308 @@ Here's an outline of the contribution process:
 7. Sign a Contributor's Agreement, if you haven't already.
 8. Your branch will be merged!
 
+### Permalinks
+
+Issues can be enhanced by using GitHub's permalink features for
+[files](https://help.github.com/articles/getting-permanent-links-to-files/) and
+[code snippets](https://help.github.com/articles/creating-a-permanent-link-to-a-code-snippet/).
```

Comment:
That's cool. I didn't know they could be used in Markdown. I have logged the issue here so that it can be tackled more holistically. https://github.com/artefactual/archivematica/issues/1092